### PR TITLE
Span Links improvements

### PR DESCRIPTION
**Pull request by tushar-datadog**
### What does this PR do? What is the motivation?
We recently released support for bi drectional span links. This PR removes the word Beta from various places and adds a newer representation for span links

### CODE REVIEW

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 1]
```diff
@@ -12,17 +12,25 @@ further_reading:
...
 
 Span links correlate one or more spans together that are causally related but don't have a typical parent-child relationship. These links may correlate spans within the same trace or across different traces.
 
 Span links help trace operations in distributed systems, where workflows often deviate from linear execution patterns. They are useful to trace the flow of operations in systems that execute requests in batches or process events asynchronously.
 
+Datadog supports both forward and backward span links, allowing users to visualize and navigate span relationships across traces in both directions.
+
+Forward Links: A span can be linked to another span that occurs later in time, whether it belongs to the same trace or a different one. This enables you to navigate from earlier operations to subsequent ones across traces.
+
+Backward Links: Similarly, a span can be linked to a span that occurred earlier in time, either within the same trace or across different traces. This allows you to trace back from later operations to previous ones.
```
Comment (hestonhoffman) [inline content/en/tracing/trace_collection/span_links/_index.md:32]:
```suggestion
- Forward Links: A span can be linked to another span that occurs later in time, whether it belongs to the same trace or a different one. This enables you to navigate from earlier operations to subsequent ones across traces.
- Backward Links: Similarly, a span can be linked to a span that occurred earlier in time, either within the same trace or across different traces. This allows you to trace back from later operations to previous ones.
```

File: content/en/tracing/trace_explorer/trace_view.md [hunk 2]
```diff
@@ -211,15 +211,15 @@ View [Profiles][1] to identify lines of code related to performance issues. The
 [1]: /profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces
 
 {{% /tab %}}
-{{% tab "Span Links (Beta)" %}}
+{{% tab "Span Links" %}}
 
 <div class="alert alert-info">Span link support is in beta.</div>
```
Comment (hestonhoffman) [inline content/en/tracing/trace_explorer/trace_view.md:216]:
```suggestion
```
Guessing this should be removed as well?

**Review by hestonhoffman — COMMENTED** [review]
Couple small suggestions for this one

File: content/en/tracing/trace_explorer/trace_view.md (hunk 2 above)
Comment (tushar-datadog) [reply content/en/tracing/trace_explorer/trace_view.md:216]: Yes!

File: content/en/tracing/trace_collection/span_links/_index.md [hunk 3]
```diff
@@ -77,4 +84,4 @@ You can view span links from the [Trace Explorer][4] in Datadog.
 [6]: https://opentelemetry.io/docs/specs/otel/trace/sdk/
 [7]: https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/
 [8]: https://docs.datadoghq.com/opentelemetry/interoperability/otlp_ingest_in_the_agent
-
+[9]: https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/?tab=datadogapi
```
Comment (brett0000FF) [inline content/en/tracing/trace_collection/span_links/_index.md:87]:
```suggestion
[9]: /tracing/trace_collection/custom_instrumentation/?tab=datadogapi
```

**Pull request merged**